examples/reuploading_classifier/main.py

```diff
@@ -25,8 +25,8 @@ def main(dataset, layers):
             # Load previous results. Have we ever run these problem?
             data = pickle.load(f)
     except:
-        print('new data')
         data = {}
+        data[dataset] = {}
```

Comment:
```suggestion
        data = {dataset: {}}
```

Reply:
@AdrianPerezSalinas here a even shorter version, so you can remove the first `data = {}`.